packages/requests/src/lib/requests-cache.ts

```diff
@@ -68,9 +68,7 @@ export function updateRequestsCache<S extends RequestsCacheState>(
 
   if (value) {
     normalized = requestsOrKeys.reduce((acc: any, key: string) => {
-      acc[key] = {
-        value,
-      };
+      acc[key] = { ...value };
```

Comment:
There is no need to use spread. `acc[key] = value`

Reply:
updated PR